operator/config/mapkubeapis/Map.yaml

```diff
@@ -0,0 +1,203 @@
+mappings:
```

Comment:
Could you move this package to `operator/pkg/config`?

Reply:
Done